corruption.js

```diff
@@ -41,17 +50,18 @@ window.addEventListener('load', function () {
     const corruptionInput = document.getElementById('corruption');
     const textInput = document.getElementById('text');
     const result = document.getElementById('result');
+    result.innerText = textInput.value.toLowerCase();
 
     let speed = speedInput.value;
-    let corruption = corruptionInput.value;
+    let corruption = 1;//corruptionInput.value;
```

Comment:
Why do you default it to 1? Just edit the HTML default value or set `corruptionInput.value = 1;`.

Reply:
Oh, and fix it so you can edit the text below.